IntegrationTests/UI.IntegrationTests/Script/ScriptRunnerTests.cs

```diff
@@ -24,6 +26,8 @@ public class ScriptRunnerTests
         // Created once for the fixture
         private ReferenceRepository _referenceRepository;
 
+        private static MethodInfo s_miRunScript = typeof(ScriptRunner).GetMethod("RunScript", BindingFlags.NonPublic | BindingFlags.Static);
```

Comment:
😉 Have you coded too much in C++ recently?
```suggestion
        // new kind of test accessor in Hungarian Notation
        private static MethodInfo s_miRunScript = typeof(ScriptRunner).GetMethod("RunScript", BindingFlags.NonPublic | BindingFlags.Static);
```
Anyways, a comment would have helped me and will do other readers.

Reply:
Hahah, nope, I can barely read C++ let alone write anything in it.
But... I've been spending a lot of time dealing with tests (mostly instabilities 😞), and my colleague developed (yet another!)  test accessor pattern (https://github.com/dotnet/winforms/pull/3225) that our codebase will benefit from too.

As far as notation is concerned - it is what we use in dotnet repos, `s_` indicates static fields.

Reply:
I put a different comment, please let me know if it needs further clarification

Reply:
With the prefix `s_`, you preach to the converted as for me -- also with `s_mi`. Though the C#/.net coding styles frowned upon.
The suggested comment was in good fun. Your comment is good. I understood the code without it. But it took some time to get the intention "being able to declare the tested method `private`".

Reply:
I plan to port more dotnet goodness when we move further in to .NET Core :)